captcha-alternative.html

```diff
@@ -0,0 +1,103 @@
+
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
```

Comment:
Please remove internal styling if we need so please add this on external style file